src/Cli/dotnet/Program.cs

```diff
@@ -29,55 +30,91 @@ public static int Main(string[] args)
         {
             DebugHelper.HandleDebugSwitch(ref args);
 
-            new MulticoreJitActivator().TryActivateMulticoreJit();
+            // Capture the current timestamp to calculate the host overhead.
+            DateTime mainTimeStamp = DateTime.Now;
 
-            if (Env.GetEnvironmentVariableAsBool("DOTNET_CLI_CAPTURE_TIMING", false))
+            // Determine if the perf log is enabled.
+            bool perfLogEnabled = Env.GetEnvironmentVariableAsBool("DOTNET_CLI_PERF_LOG", false);
+            PerformanceLogStartupInformation startupInfo = null;
+            if (perfLogEnabled)
             {
-                PerfTrace.Enabled = true;
-            }
+                // Initialize the performance log start-up information.
+                startupInfo = new PerformanceLogStartupInformation(mainTimeStamp);
 
-            InitializeProcess();
+                // Initialize the performance log manager and kick off asynchronous clean-up.
```

Comment:
nit: maybe rename PerformanceLogManager.Initialize -> PerformanceLogManager.InitializeAndKickOffAsynCleanUp? 

It is a method already. So we can also remove one comment. Comments have maintenance cost.

Reply:
Done.